website/docs/using-plugins.md

```diff
@@ -458,3 +458,78 @@ import thumbnail from './path/to/img.png';
 | `max` | `integer` |  | See `min` above |
 | `steps` | `integer` | `4` | Configure the number of images generated between `min` and `max` (inclusive) |
 | `quality` | `integer` | `85` | JPEG compression quality |
+
+### `@docusaurus/plugin-client-redirects`
+
+Docusaurus Plugin to generate **client-side redirects**.
+
+This plugin will write additional HTML pages to your static site, that redirects the user to your existing Docusaurus pages with Javascript.
+
+:::caution
+
+It is better to use server-side redirects whenever possible.
+
+Before using this plugin, you should look if your hosting provider doesn't offer this feature.
+
+As far as we know, only Github Pages does not offer server-side redirects.
```

Comment:
I think it’s better to remove this sentence, as it is quite specific. In the future, GitHub Pages may implement redirect support, and we will forget to update the docs (just an example).

Reply:
@lex111 only the last one right? agree

Reply:
Yes